docs/content/authorization-concepts.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Authorization Concepts
+description: Introduction to Authorization 
+sidebar_position: 1
+slug: /authorization-concepts
+---
+
+import { ProductName, ProductNameFormat, RelatedSection } from '@components/Docs';
+
+# Authorization Concepts 
+
+## Authentication vs Authorization
+
+[**Authentication**](https://auth0.com/intro-to-iam/what-is-authentication/) (or **AuthN**) ensures a user's identity. [**Authorization**](https://en.wikipedia.org/wiki/Authorization) (or **AuthZ**) determines if a user can perform a certain action on a particular resource.
+
+For example, when you log in to Google, Authentication is the process of verifying that your username and password are correct. Authorization is the process of ensuring that you can access a given Google service or feature. [For more information about AuthN vs AuthZ, click here.](https://www.okta.com/identity-101/authentication-vs-authorization/)
```

Comment:
Don't use "click here" for hyperlinks. @pavokta: https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544471330/Other+Conventions#Ensure-link-text-is-descriptive-of-its-destination